common/automine_after_reboot.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=the indirect restart of automine after a bad gpu caused a reboot
+Requires=automine.service
+After=automine.service
+
+[Service]
+ExecStartPre=/bin/mkdir -p /tmp/automine
+ExecStart=/bin/bash -c "cp %h/bin/automine/overclock.json /tmp/automine/overclock.json"
+ExecStartPost=/bin/systemctl --user disable automine_after_reboot
```

Comment:
@alecrem 
FYI: this line disables the automine_after_reboot service whenever it completes successfully.  At the moment, the automine_after_reboot service only gets enabled if a cpu fails.

In the future, it should be OK to
- remove this line
- update install_systemd_units so that automine_after_reboot enabled by default
Doing so will mean that miner starts up and its overclock is applied whenever the user logs i.e, for auto login users, whenever the machine restarts.